### ISSUE TITLE
[Bug]: Unable to verify the first certificate

### ISSUE THREAD

**Issue by ttsirkia** [RapidAPI/feedback#88]
### What happened?

After updating the plugin, I'm not able to run all requests anymore. They fail with the following info in the log:

```
unable to verify the first certificate
```

The SSL verification option does not have any effect on this. I downgraded back to version 1.8.2 and it is working again.

### Steps to reproduce?

Run a query to server which has more exotic SSL certificate.

### VS Code version

1.74.3

### Extension version

1.9.2

### Relevant log output

```
["VERBOSE" - 2023-01-15T11:16:26.631Z - v1.9.2] 
  Proxy fetch: https://addressremoved - GET - [{"name":"Content-Type","value":"multipart/form-data; charset=utf-8; boundary=__X_RAPIDAPI_BOUNDARY__"}] - {"method":"POST","headers":{"Content-Type":"application/json"},"body":{"requestUuid":"61cbe041-ac5c-4b21-b511-66f41f7cbbd7","projectUuid":"vscode-generator","projectId":"f5809ef1-75c1-4efd-81a5-ced8b24f4b6a","sessionId":"5986882a-d402-41e6-86d0-e5c5abb9a1cf","storeCookies":true,"libName":"RapidAPI - VS Code","request":{"method":"GET","url":"https://addressremoved","params":{},"headers":{"Content-Type":"multipart/form-data; charset=utf-8; boundary=__X_RAPIDAPI_BOUNDARY__"},"bodyType":"text","data":"\r\n--__X_RAPIDAPI_BOUNDARY__--\r\n"}},"mode":"cors","credentials":"same-origin","cache":"no-cache","redirect":"manual"}
["VERBOSE" - 2023-01-15T11:16:26.637Z - v1.9.2] 
  RapidAPI Client started: 54
["ERROR" - 2023-01-15T11:16:26.763Z - v1.9.2] 
  Proxy fetch ERROR: "request to https://addressremoved failed, reason: unable to verify the first certificate"
```

**Comment by reidsneo** [comment]
+1 I also have same issue

**Comment by estruyf** [comment]
@ttsirkia @reidsneo, thanks for opening an issue around this. We'll try to get this reproduced and fixed.

Do you have APIs you can share which are causing the issue?

**Comment by ttsirkia** [comment]
Unfortunately my observation is from API which is not publicly available. I can check if I could easily find some open APIs having the same issue.

**Comment by ttsirkia** [comment]
Just send GET request to https://self-signed.badssl.com/

**Comment by ttsirkia** [comment]
Or alternatively https://untrusted-root.badssl.com/

**Comment by estruyf** [comment]
Thanks, we added a fixed and verified it with the given URLs.

![Screenshot 2023-01-19 at 14 17 50](https://user-images.githubusercontent.com/2900833/213453107-3fa11e08-2662-41ae-a994-cc4544ca193b.png)

![Screenshot 2023-01-19 at 14 18 00](https://user-images.githubusercontent.com/2900833/213453131-2bc367d7-21fd-4803-bb44-e77861b2e08b.png)

**Comment by estruyf** [comment]
Version `1.9.3` is coming out in a couple of minutes.

**Comment by ttsirkia** [comment]
@estruyf thank you very much!

**Comment by estruyf** [comment]
Let me know if this version works for you so we can close the issue.

Thank you for reporting this issue! 🙏

**Comment by ttsirkia** [comment]
I try and if successful, I will close the issue.

**Comment by ttsirkia** [comment]
Fixed in version 1.9.3.

**Comment by reidsneo** [comment]
thank you also solved my issue! +1
